cura/Settings/ExtruderManager.py

```diff
@@ -513,17 +514,32 @@ def getActiveExtruderStacks(self) -> List["ExtruderStack"]:
         global_stack = Application.getInstance().getGlobalContainerStack()
 
         result = []
+        machine_extruder_count = global_stack.getProperty("machine_extruder_count", "value")
+
+        if machine_extruder_count is 1:
```

Comment:
is shouldn't be used in this case; should be == 
Also; document why we are aborting already.